d2renderers/d2svg/d2svg.go

```diff
@@ -113,6 +113,15 @@ func arrowheadDimensions(arrowhead d2target.Arrowhead, strokeWidth float64) (wid
 	case d2target.DiamondArrowhead:
 		widthMultiplier = 11
 		heightMultiplier = 9
+	case d2target.CfOne:
+		fallthrough
+	case d2target.CfMany:
+		fallthrough
+	case d2target.CfOneRequired:
+		fallthrough
+	case d2target.CfManyRequired:
```

Comment:
let's use commas instead of fallthroughs, little less words

Reply:
done.